Account/FirefoxAccountConfiguration.swift

```diff
@@ -72,7 +75,7 @@ public struct LatestDevFirefoxAccountConfiguration: FirefoxAccountConfiguration
 }
 
 public struct StableDevFirefoxAccountConfiguration: FirefoxAccountConfiguration {
-    public init() {
+    public init(prefs: Prefs? = nil) {
```

Comment:
This change isn't needed if you remove the constructor from the `FirefoxAccountConfiguration`.